relay-dynamic-config/src/project.rs

```diff
@@ -91,6 +97,7 @@ impl Default for ProjectConfig {
             breakdowns_v2: None,
             session_metrics: SessionMetricsConfig::default(),
             transaction_metrics: None,
+            metric_extraction: Default::default(),
```

Comment:
`Default` is implemented in the same PR by invoking `MetricExtractionConfig::default()`. I prefer to avoid an option here since it's just not needed -- the default state is disabled, so the option is redundant.